README.md

```diff
@@ -242,6 +242,8 @@ Available in [latest browsers](http://caniuse.com/#feat=intersectionobserver). I
 
 For IE11 support, please make use of these [polyfills](https://polyfill.io/v3/polyfill.min.js?flags=gated&features=Object.assign%2CIntersectionObserver).
```

Comment:
@jprats89 instead of adding another line, better update the existing `polyfill.io` bundle link to include this function, also, test once.

Reply:
Hi there.

I tried to update the *polyfill.io* bundle, but I could not find a pollyfill in that platform for ParentNode.append(). I will try to do some research to understand why during the weekend

Reply:
Well, definetly ParentNode is not included in Pollyfill.io. Searching in their [project issues](https://github.com/Financial-Times/polyfill-service/issues?utf8=%E2%9C%93&q=ParentNode) there are several request to include it but no one did yet.

I know this solution of a new line in the readme is a dirty one, but I sincerely do not know how to improve it.

Best solution would be a PR to pollyfill.io including the ParentNode pollyfill ir replace the use of ParentNode in this project by other supported class but my that's out of my scope in term of JS knowlege.

If you have any idea just let me know and I can try to carry it out